### PR TITLE
Handle child refs

### PR DESCRIPTION
**Pull request by airtoxin**
ref https://github.com/Yomguithereal/baobab-react/pull/104

Thanks base PR @matthewkturner
I can't wait for releasing that feature so I wrote test.

### CODE REVIEW

**Comment by matthewkturner** [comment]
Thanks @airtoxin!

File: src/higher-order.js [hunk 1]
```diff
@@ -78,6 +78,14 @@ function branch(cursors, Component) {
 
   const ComposedComponent = class extends React.Component {
 
+    getDecoratedComponentInstance() {
+	return this.decoratedComponentInstance
```
Comment (Yomguithereal) [inline src/higher-order.js]: Just add a semicolon here

File: src/higher-order.js [hunk 2]
```diff
@@ -78,6 +78,14 @@ function branch(cursors, Component) {
...
+    }
+
+    handleChildRef(component) {
+	this.decoratedComponentInstance = component
```
Comment (Yomguithereal) [inline src/higher-order.js]: ... and here and we're good to merge :)

**Comment by airtoxin** [comment]
fixed!

**Comment by Yomguithereal** [comment]
Thanks @airtoxin. Merging right now. I don't have time right now but if you want to add some documentation about this in a PR, I'll gladly merge it too. Thanks!

**Comment by Yomguithereal** [comment]
Just published a `2.1.0` with your additions.

**Pull request merged**